File: src/com/jsoup/scrapping/Resource.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.jsoup.scrapping;

/**
 *
 * @author Sabbir
 */
public class Resource {
    private String tag;
    private String text;
    private MediaResource media;

    public Resource(String tag, String text, MediaResource media) {
        this.tag = tag;
        this.text = text;
        this.media = media;
    }

    public String getTag() {
        return tag;
    }

    public String getText() {
        return text;
    }

    public MediaResource getMedia() {
        return media;
    }
    
    
    
}

File: src/com/jsoup/scrapping/Resource.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.jsoup.scrapping

import java.lang.StringBuilder

/**
 *
 * @author Sabbir
 */
data class Resource(
        var tagName: String?,
        var resourceText: StringBuilder?,
        var mediaName: String?,
        var mediaSrc: String?,
        var mediaHref: String?,
        var mediaheight: Any?,
        var mediawidth: Any?
) {
    constructor() : this(
            null,
            null,
            null,
            null,
            null,
            null,
            null
    )
}